packages/flame/test/extensions/_help.dart

```diff
@@ -0,0 +1,6 @@
+mixin NullableEqualsMixin on Object {
+  @override
+  bool operator ==(Object? other) {
```

Comment:
I don't think that this does anything? You can already use `==` on null values.

Reply:
Aah, that makes sense, seems like a bug from `Mock`'s side to not include null though.